percona-distribution-postgresql-11/README.md

```diff
@@ -18,7 +18,7 @@ Images are updated when new releases are published.
 
 Start a Percona Distribution for PostgreSQL container as follows:
 
-    docker run --name container-name -d perconalab/percona-distribution-postgresql:tag
+    docker run --name container-name -e POSTGRESQL-PASSWORD=secret -d perconalab/percona-distribution-postgresql:tag
```

Comment:
The variable uses an underscore instead of a hyphen:
```
POSTGRESQL-PASSWORD=secret
```
should be
```
POSTGRESQL_PASSWORD=secret
```
Perhaps it would be nice to show the better options here too (secrets or `--env-file`)?

Reply:
thanks for the comment and review. Updated the variable and added the section for using the env-file